examples/svelte-kitchen-sink/jest.config.js

```diff
@@ -1,11 +1,12 @@
 const config = require('../../jest.config');
 
 module.exports = {
+  verbose: true,
```

Comment:
Do we want this on?

Reply:
oops my bad, I was debugging the jest transform and I forgot to remove it.